phimpme.sh

```diff
@@ -0,0 +1,20 @@
+mkdir $HOME/openCV/
+cd $HOME/openCV
+wget https://github.com/opencv/opencv/releases/download/4.0.1/opencv-4.0.1-android-sdk.zip
+
+unzip opencv-4.0.1-android-sdk.zip
+rm -rf opencv-4.0.1-android-sdk.zip
+# Rename folder name to opencv
+mv OpenCV-android-sdk opencv
+
+cd $TRAVIS_BUILD_DIR
```

Comment:
You should absolutely have no dependency on TRAVIS at all. That is the entire point of this issue. To create a script which downloads and sets up OpenCV WITHOUT any environment dependency on Travis

The comment literally says, should work in both Travis and Fdroid. How can it work in fdroid if it depends on TRAVIS_BUILD_DIR

Reply:
@iamareebjamal to provide the path to phimpme-android, I need to provide the travis path. How else can I copy the openCV files?

Reply:
You answered it yourself in next commit